react/src/lib/components/DeckGLMap/layers/wells/wellsLayer.stories.tsx

```diff
@@ -482,16 +482,22 @@ const reverseRange = false;
 //eslint-disable-next-line
 const wellLayerTemplate = (args: any) => {
     const [getColorName, setColorName] = React.useState("Rainbow");
-
+    const [isLog, setLog] = React.useState(false);
```

Comment:
normally, we just put set as the prefix of the variable name so I guess setIsLog will be more appropriate here to indicate that this is the state action of the vairable isLog